pkgs/by-name/wg/wg-access-server/package.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, buildGoModule
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "wg-access-server";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "freifunkMUC";
+    repo = "wg-access-server";
+    rev = "v${version}";
+    hash = "sha256-m7Zf8LAkVgFHBNgCr8TZTtp28LH2tFRLhdpulp//pbM=";
+  };
+
+  proxyVendor = true; # darwin/linux hash mismatch
+  vendorHash = "sha256-aEJu3dTKyu2d/xI03DEeOqNl0nkAPkURF/AK7mOf5A4=";
+
+  CGO_ENABLED = 1;
+
+  ldflags = [ "-s" "-w" ];
+
+  doCheck = false;
```

Only one package (`internal/dnsproxy`) fails tests due to requiring network access, other tests seem to pass OK.

How about something like
```suggestion
  checkFlags = [ "-skip=TestDNSProxy_ServeDNS" ];
```